Makefile

```diff
@@ -3,8 +3,11 @@ MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rules
 
 PYTHON := python
-BUILD_SCRIPTS_LOCATION := ${PWD}/building
-BUILD_SCRIPTS_LOCATION_RELATIVE := building
```

Comment:
This is an example of the absolute vs. relative location thing. I don't think both are necessary as long as we're consistent about it throughout the makefile. If we use relative in one instance but absolute in another, the prerequisite rule doesn't get picked up and it errors.

Reply:
Hm it looks like BUILD_SCRIPTS_LOCATION_RELATIVE is vestigial, it's not used.

Reply:
I see. I've definitely switched to using it a lot in this branch. On top of that, all handwritten file references are relative (because, of course, they have to be). I'd prefer to make every location reference relative instead of absolute for consistency's sake, and minimize use of `${PWD}` as much as possible.

The long grep command a couple dozen lines below this, and the rules it sets up, might not be compatible with this though.